robot/rospackages/src/mcu_control/scripts/PdsNode.py

```diff
@@ -16,7 +16,7 @@
 from sensor_msgs.msg import JointState
 from mcu_control.srv import *
 
-global ser  # make global so it can be used in other parts of the code
+ser = None # Initialise variable for future use
```

Comment:
you might need to keep this shit variable global since it has to be modified somewhere else.

Reply:
This thing is changed in #282